lib/contentful/query.ex

```diff
@@ -444,6 +449,11 @@ defmodule Contentful.Query do
     Contentful.Stream.stream(args, space, env, api_key)
   end
 
+  @doc """
+  Returns the filter modifiers supported byt the Query syntax
+  """
+  @spec allowed_filter_modifiers() :: list()
+  @since "0.4.0"
```

Comment:
So, should this be either `@doc since: "0.4.0"` or `@since "0.4.0"`? Both are used in this file. I'm confused.

Reply:
José says, we should prefer `@doc since: "0.4.0"`; https://github.com/elixir-lang/elixir/issues/7830.
It should be okay to have duplicated `@doc` attributes, as long as only one is the binary/text.

Reply:
Don't code while walking through the park i guess :D